SupportClient/SupportClient/Model/Feedback.swift

```diff
@@ -6,11 +6,13 @@
 //  Copyright © 2020 Cornell Appdev. All rights reserved.
 //
 
-// TODO: replace this class once we hook up to database
+
+
 struct Feedback: Codable {
     
-    var title: String
+    var has_read: Bool
     var message: String
-    var hasRead: Bool
+    var title: String
+    var twoway: Bool
```

Comment:
are we using `FutureNova` for networking? if so we can do
```
        Endpoint.config.keyDecodingStrategy = .convertFromSnakeCase
        Endpoint.config.keyEncodingStrategy = .convertToSnakeCase
```
otherwise, we should still be able to specify how we do the decoding and encoding so that we can stick to camelCase naming for variables even if the server returns snake case